packages/server/src/api/controllers/static/index.ts

```diff
@@ -145,6 +146,16 @@ const requiresMigration = async (ctx: Ctx) => {
 }
 
 export const serveApp = async function (ctx: UserCtx) {
+  if (ctx.url.includes("apple-touch-icon")) {
```

Comment:
Should we be more specific here? e.g. `ctx.url.includes("/apple-touch-icon.png")`?